lib/simp/spec_helpers/version.rb

```diff
@@ -1,5 +1,5 @@
 module Simp; end
 
 module Simp::SpecHelpers
-  VERSION = '0.0.1'
+  VERSION = '0.0.2'
```

Comment:
This patch adds new API functionality.

```suggestion
  VERSION = '0.1.0'
```